js/src/components/AnalysisUpsell.js

```diff
@@ -56,7 +56,7 @@ const AnalysisUpsell = ( props ) => {
 				) }
 			</TextContainer>
 			<div>
-				<OutboundLinkButton href={ url }>
+				<OutboundLinkButton href={ url } className={ "UpsellLinkButton" }>
```

Comment:
Hopefully this does not affect the styling outside of Elementor. Will test in ACC

Reply:
Class will be present outside of Elementor. But it wont' have this styling, as `elementor.css` isn't enqueued there. CSS is copied from the block-editor btw, so even when enqueued it would just be a duplicate.